tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,402 @@
+	}
+
+	/**
+	 * Get the set currency symbol.
+	 *
+	 * @return string|void
+	 */
+	protected function get_current_symbol() {
+		$post_id = Tribe__Main::post_id_helper();
+
+		// first check $_POST, else check post meta
+		if ( isset( $_POST['EventCurrencySymbol'] ) ) {
+			$currency_symbol = $_POST['EventCurrencySymbol'];
+		} else {
+			$currency_symbol = tribe_get_event_meta( $post_id, '_EventCurrencySymbol', true );
+		}
+
+		return esc_attr( $currency_symbol );
+	}
+
+	/**
+	 * Get the set currency symbol position.
+	 *
+	 * @return string|void
+	 */
+	protected function get_current_symbol_position() {
+		$post_id = Tribe__Main::post_id_helper();
+
+		// first check $_POST, else check post meta
+		if ( isset( $_POST['EventCurrencyPosition'] ) ) {
+			$prefix_suffix = $_POST['EventCurrencyPosition'];
+		} else {
+			$prefix_suffix = tribe_get_event_meta( $post_id, '_EventCurrencyPosition', true );
+		}
+
+		return esc_attr( $prefix_suffix );
+	}
+
+	/**
+	 * Get the default currency position
+	 *
+	 * @return string
+	 */
+	protected function get_symbol_position_default() {
+		$default_position = tribe_get_option( 'reverseCurrencyPosition' ); // boolean
+		if ( ! empty( $default_position ) ) {
+			$default_position = 'suffix';
+		} else {
+			$default_position = 'prefix';
+		}
+
+		$default_position = apply_filters( 'tribe_ext_ce_cost_currency_symbol_position_default', $default_position );
+
+		return sanitize_text_field( $default_position );
+	}
+
+	/**
+	 * Get the symbol position phrase
+	 *
+	 * @param string $position
+	 *
+	 * @return string
+	 */
+	protected function get_symbol_position_phrase( $position = 'prefix' ) {
+		if ( 'prefix' === $position ) {
+			$output = esc_attr_x( 'Before Cost', 'Currency symbol position', 'tribe-extension' );
+		} else {
+			$output = esc_attr_x( 'After Cost', 'Currency symbol position', 'tribe-extension' );
+		}
+
+		return $output;
+	}
+
+	/**
+	 * Build the list of select options for the allowed currency symbols.
+	 *
+	 * @return string
+	 */
+	protected function symbol_list_select_options() {
+		$symbol_list_array = $this->symbol_list_array();
+
+		if ( empty( $symbol_list_array ) ) {
+			return '';
+		}
+
+		$current_symbol = (string) $this->get_current_symbol();
+
+		$post_id = Tribe__Main::post_id_helper();
+
+		/**
+		 * If currency symbol is not set and we verify we are on the Add New
+		 * Event version of the Community Events form, then set the currency
+		 * symbol to the default setting. This avoids setting a purposefully-blank
+		 * currency symbol to the default symbol when on the Edit Event version of
+		 * the Community Events form.
+		 */
+		if (
+			'' === $current_symbol
+			&& $this->is_community_page( 'add-event' )
+		) {
+			$current_symbol = tribe_get_option( 'tribe_ext_ce_cost_currency_symbol_default' );
+		}
+
+		$output = sprintf( '%1$s<option value=""></option>%1$s', PHP_EOL );
+
+		foreach ( $symbol_list_array as $key => $value ) {
+			$value = esc_attr( $value );
```

Comment:
I commented, deleted, and am not re-commenting after going back and forth with my thoughts on this one.

Long story short, if I don't escape initially, then I also have to escape `$selected`, since not-yet-escaped `$value` is also used there. So that's what I did.